storops/unity/resource/filesystem.py

```diff
@@ -80,7 +80,7 @@ def create(cls, cli, pool, nas_server, name, size, proto=None,
     def modify(self, size=None, is_thin=None, tiering_policy=None,
                user_cap=False, is_compression=None, access_policy=None,
                locking_policy=None, description=None,
-               cifs_fs_parameters=None):
+               cifs_fs_parameters=None, snapScheduleParameters=None):
```

Comment:
snapScheduleParameters need to change to snap_schedule_parameters

Reply:
Done

Reply:
Done